tests/models/test_licensing.py

```diff
@@ -554,53 +554,70 @@ def test_set_presentation_edition(self):
 
-        # set overlapping fields on editions
-        edition_od.title = u"OverdriveTitle1"
+        admin = self._edition(data_source_name=DataSource.LIBRARY_STAFF, with_license_pool=False)
+        admin.primary_identifier = pool.identifier
 
-        edition_mw.title = u"MetadataWranglerTitle1"
-        edition_mw.subtitle = u"MetadataWranglerSubTitle1"
+        mw = self._edition(data_source_name=DataSource.METADATA_WRANGLER, with_license_pool=False)
+        mw.primary_identifier = pool.identifier
 
-        edition_admin.title = u"AdminInterfaceTitle1"
+        # The library staff has no opinion on the book's medium,
+        # and the metadata wrangler has an incorrect opinion.
+        admin.medium = None
+        mw.medium = Edition.BOOK_MEDIUM
 
-        pool.set_presentation_edition()
-
-        edition_composite = pool.presentation_edition
+        # Overdrive, the metadata wrangler, and the library staff all have
+        # opinions on the book's title. The metadata wrangler has also
+        # identified a subtitle.
+        od.title = u"OverdriveTitle1"
 
-        assert_not_equal(edition_mw, edition_od)
-        assert_not_equal(edition_od, edition_admin)
-        assert_not_equal(edition_admin, edition_composite)
-        assert_not_equal(edition_od, edition_composite)
+        mw.title = u"MetadataWranglerTitle1"
+        mw.subtitle = u"MetadataWranglerSubTitle1"
 
-        # make sure admin pool data had precedence
-        eq_(edition_composite.title, u"AdminInterfaceTitle1")
-        eq_(edition_admin.contributors, edition_composite.contributors)
+        admin.title = u"AdminInterfaceTitle1"
 
-        # make sure data not present in the higher-precedence editions didn't overwrite the lower-precedented editions' fields
-        eq_(edition_composite.subtitle, u"MetadataWranglerSubTitle1")
-        [license_pool] = edition_composite.is_presentation_for
-        eq_(license_pool, pool)
-
-        # Change the admin interface's opinion about who the author
-        # is.
-        for c in edition_admin.contributions:
+        # Create a presentation edition, a composite of the available
+        # Editions.
+        pool.set_presentation_edition()
+        presentation = pool.presentation_edition
+        eq_([license_pool], presentation.is_presentation_for)
+
+        # The presentation edition is a completely new Edition.
+        assert_not_equal(mw, od)
+        assert_not_equal(od, admin)
+        assert_not_equal(admin, presentation)
+        assert_not_equal(od, presentation)
+
+        # Within the presentation edition, information from the
+        # library staff takes precedence over anything else.
+        eq_(presentation.title, u"AdminInterfaceTitle1")
+        eq_(admin.contributors, presentation.contributors)
+
+        # Where the library staff has no opinion, the license source
+        # takes precedence over the metadata wrangler.
+        eq_(Edition.AUDIO_MEDIUM, presentation.medium)
```

Comment:
Similar here -- this is the only new bit, everything else in this test is cleanup.